### DISCUSSION TITLE
New Issue format

### DISCUSSION THREAD

**Discussion by inmanturbo** [livewire/livewire#6319] [4. General]
The only issue I am seeing with not using issues (lol), is that it is very hard to tell which ones have been resolved. When I face a bug, I check for issues. If there are none open, I look through the closed ones to see if my issue has already been resolved in some way, before posting. Also I check in on issues from time to time to keep track of progress. Which bugs have been worked on? Which have been fixed? If they are all closed I can assume most raised bugs were resolved, and I don't have to drill down too far into very many of them. 

Discussions is a beast. There are tons of bugs being reported. I have no idea which were resolved, or otherwise dismissed  as "not really a bug" without drilling down into each and every one. Even the reply/(click to expand) format requires even more drilling down and down to get to the heart of the matter. Also OP's often don't get around to marking them as "answered".

I don't feel confident in the system. lol. Maybe because I  am not accustomed to it. Since they don't need to be closed, and there is high potential for spam, how many are actually reviewed, and how well are they actually tracked?

Am I just looking at this all wrong? How should I best use the discussions system? What's a good "flow"?

**Comment by PerryvanderMeer** [comment]
I definitely agree with this. Personally, I find the 'Discussions' section very confusing. The original reasons for moving from issues to discussions were included in https://github.com/alpinejs/alpine/discussions/1423 two years ago. To address the reasons given then:

> People treat them as a dumping ground for bugs, feature requests, questions, etc.... and because of this they require extra triaging.

In my opinion, this is exactly what is happening in the `Discussions' section right now.

> Embedded in them is the concept of being "open" or "closed" which is problematic. Many people feel "wronged" when an issue is closed and interpret that action as an act of aggression from the maintainer.

I do understand this point, but this is exactly what makes the 'Discussions' section so confusing. There are nearly 3,000 discussions, whose status is unknown. With nearly a million downloads per month, it is sometimes necessary to be 'aggressive' (rather clear in my opinion).

In the past two years there have been many changes to the 'Issues' section: forms, projects, milestones, labels, responding with emojis, etc. In the end, the whole infrastructure hangs together from issues and pull requests. Is there any possibility that the choice made will be reconsidered? I think it benefits development in the end.

@calebporzio @joshhanley
